launch/tier4_localization_launch/launch/pose_twist_estimator/pose_twist_estimator.launch.xml

```diff
@@ -21,7 +21,9 @@
         <arg name="sub_gnss_pose_cov" value="/sensing/gnss/pose_with_covariance"/>
       </include>
       <include file="$(find-pkg-share automatic_pose_initializer)/launch/automatic_pose_initializer.launch.xml"/>
-      <include file="$(find-pkg-share tier4_localization_launch)/launch/util/util.launch.py"/>
+      <include file="$(find-pkg-share tier4_localization_launch)/launch/util/util.launch.py">
+        <arg name="input/pointcloud" value="$(var input_pointcloud)"/>
```

Comment:
How about also modifying `input/pointcloud` in `util.launch. py` ? 
Then we don't need to write this line.

Reply:
Thanks for your suggestion. I agree, it looks like it would be better this way. I made necessary changes. Thank you.